internal/cli/atlas/serverless/backup/restores/create.go

```diff
@@ -0,0 +1,216 @@
+}
+
+func (opts *CreateOpts) isPointInTimeRestore() bool {
+	return opts.deliveryType == pointInTimeRestore
+}
+
+func (opts *CreateOpts) isDownloadRestore() bool {
+	return opts.deliveryType == downloadRestore
+}
+
+func markRequiredAutomatedRestoreFlags(cmd *cobra.Command) error {
+	if err := cmd.MarkFlagRequired(flag.TargetProjectID); err != nil {
+		return err
+	}
+
+	if err := cmd.MarkFlagRequired(flag.SnapshotID); err != nil {
+		return err
+	}
+
+	if err := cmd.MarkFlagRequired(flag.TargetClusterName); err != nil {
+		return err
+	}
+
+	return cmd.MarkFlagRequired(flag.ClusterName)
+}
+
+func markRequiredPointInTimeRestoreFlags(cmd *cobra.Command) error {
+	if err := cmd.MarkFlagRequired(flag.TargetProjectID); err != nil {
+		return err
+	}
+
+	return cmd.MarkFlagRequired(flag.TargetClusterName)
+}
+
+// atlas serverless backup(s) restore(s) job(s) create <automated|download|pointInTime>.
+func CreateBuilder() *cobra.Command {
+	opts := new(CreateOpts)
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Start a restore job for your serverless instance.",
+		Long: `If you create an automated or pointInTime restore job, Atlas removes all existing data on the target cluster prior to the restore.
+
+` + fmt.Sprintf(usage.RequiredRole, "Project Owner"),
+		Args: require.NoArgs,
+		Example: fmt.Sprintf(`  # Create an automated restore:
+  %[1]s serverless backup restore create \
+         --deliveryType automated \
+         --clusterName myDemo \
+         --snapshotId 5e7e00128f8ce03996a47179 \
+         --targetClusterName myDemo2 \
+         --targetProjectId 1a2345b67c8e9a12f3456de7
+
+  # Create a point-in-time restore:
+  %[1]s serverless backup restore create \
+         --deliveryType pointInTime \
+         --clusterName myDemo \
+         --pointInTimeUTCMillis 1588523147 \
+         --targetClusterName myDemo2 \
+         --targetProjectId 1a2345b67c8e9a12f3456de7
+  
+  # Create a download restore:
+  %[1]s serverless backup restore create \
+         --deliveryType download \
+         --clusterName myDemo \
+         --snapshotId 5e7e00128f8ce03996a47179`, cli.ExampleAtlasEntryPoint()),
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			if opts.isAutomatedRestore() {
+				if err := markRequiredAutomatedRestoreFlags(cmd); err != nil {
+					return err
+				}
+			}
+
+			if opts.isPointInTimeRestore() {
+				if err := markRequiredPointInTimeRestoreFlags(cmd); err != nil {
+					return err
+				}
+			}
+
+			if opts.isDownloadRestore() {
+				if err := cmd.MarkFlagRequired(flag.SnapshotID); err != nil {
+					return err
+				}
+			}
+
+			return opts.PreRunE(
+				opts.ValidateProjectID,
+				opts.initStore(cmd.Context()),
+				opts.InitOutput(cmd.OutOrStdout(), createTemplate),
+			)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return opts.Run()
+		},
+	}
+
+	cmd.Flags().StringVar(&opts.snapshotID, flag.SnapshotID, "", usage.SnapshotID)
+	// Atlas uses cluster name
```

Comment:
deja vu, I have a feeling this is copy/paste and the reason this existed was becuase the comamnd where this is coming from supports both atlas and ops manager where in ops manager we use cluster id so I think the comment can be removed